wp_bak/app/wp_bak.py

```diff
@@ -16,23 +17,18 @@
 # CONSTANTS
 #############################
 
-SRC_DIR='src/'
-DST_DIR='dst/'
-SHORT_DIR=DST_DIR+'shorts/'
-LONG_DIR=DST_DIR+'longs/'
-
-MINUTE = 60
-HOUR = 60 * MINUTE
-DAY = 24 * HOUR
-WEEK = 7 * DAY
+SRC_DIR=Path('src')
+DST_DIR=Path('dst')
+SHORT_DIR=DST_DIR.joinpath('shorts')
```

Comment:
This is fine. Just FYI that `pathlib` overloads operator `/` so `DST_DIR / 'shorts'` is the same as what you have here. I find that particularly readable since `/` is the standard directory separator but it's just a matter of opinion.